.github/workflows/_publish_badge.yaml

```diff
@@ -79,6 +79,15 @@ jobs:
               gist_id: gistId,
               files: {
                 [filename]: { content },
-                ...Object.fromEntries(Object.entries(gist.files).filter(([name]) => name !== filename))
-              }
-            });
+                ...Object.fromEntries(
+                  Object.entries(gist.files)
+                    .filter(([name]) => name !== filename)
+                    .map(([name, value]) => [
+                      name,
+                      Object.fromEntries(
+                        Object.entries(value).filter(([nestedKey]) => nestedKey === "content")
```

Comment:
Why do we only keep `content`? What are the attributes that we are trying to get rid of?

Reply:
I stumbled upon this b/c my job was failing. Here's an example: https://github.com/NVIDIA/JAX-Toolbox/actions/runs/5383094321

Then I looked up the gist api for updating a gist and apparently it says it only supports "content" and an optional "filename". So I just filtered out content, since the filename I assumed was inferred from the key.

I'm actually not sure how this worked in the first place, maybe the API was more permissive when the files within the gist were already there, but I ran into it b/c I was creating new files?